include/bluetooth/mesh/models.rst

```diff
@@ -16,11 +19,45 @@ documentation.
 
    ../../../include/bluetooth/mesh/gen_*
 
-Common model types
-===================
+For more information about these and other models, see also `Mesh Model Overview`_.
+
+.. _bt_mesh_models_categorization:
+
+Categorization
+==============
+
+Each model is classified into one of the following categories:
+
+    * Server -- which contains states.
+    * Client -- which reads and writes the the Server's states.
+
+Moreover, some models also include a *Setup Server* model instance.
+The two server model instances share the states of the server model, but accept different messages.
+This allows for a fine-grained control of the access rights for the states, as the two model instances can be bound to different application keys.
+Typically, the Setup Server instance provides write access to configuration parameters, such as valid parameter ranges or default values.
+
+Both server and client models can be extended, but because client models do not have states, there is generally no reason to extend them.
+None of the specification client models extend other models.
+All server models store changes to their configuration persistently using the :ref:`settings` subsystem.
+
+.. _bt_mesh_models_configuration:
+
+Configuration
+=============
+
+You can configure Mesh models in |NCS| using Kconfig options.
```

Comment:
You could link to :ref:`configure_application` here. ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configuring-your-application )

Reply:
Nice, thanks.